packages/webapp/pages/badges/[badgeId].tsx

```diff
@@ -0,0 +1,76 @@
+import React, { ReactElement } from 'react';
+import {
+  GetStaticPathsResult,
+  GetStaticPropsContext,
+  GetStaticPropsResult,
+} from 'next';
+import { graphqlUrl } from '@dailydotdev/shared/src/lib/config';
+import { TopReaderBadge } from '@dailydotdev/shared/src/components/badges/TopReaderBadge';
+import type { LoggedUser } from '@dailydotdev/shared/src/lib/user';
+import type { Keyword } from '@dailydotdev/shared/src/graphql/keywords';
+import { TOP_READER_BADGE_BY_ID } from '@dailydotdev/shared/src/graphql/users';
+
+export async function getStaticPaths(): Promise<GetStaticPathsResult> {
+  return { paths: [], fallback: 'blocking' };
```

Comment:
I just copied what's on the devcard page 😬 
https://github.com/dailydotdev/apps/blob/4aac500d780ef8673a8cc160bd4abb9f56101394/packages/webapp/pages/devcards/%5BuserId%5D.tsx#L11-L13

Reply:
Based on docs, `blocking` makes it so that they gets SSR. Useful for when we need to take a screenshot, and not have to wait for the data to be loaded.

Reply:
Ah might just be the specific need for these image endpoints then :)